### ISSUE TITLE
[sklearn#33945] Indicate array API support on an estimator's API page

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#371]
**Mirrored from [scikit-learn#33945](https://github.com/scikit-learn/scikit-learn/issues/33945)**

Original title: Indicate array API support on an estimator's API page
Original labels: Documentation, Needs Decision, Array API
Created: 2026-05-07T16:46:37Z

---

### Describe the issue linked to the documentation

Today I felt like it would be nice to be able to see directly on https://scikit-learn.org/stable/modules/generated/sklearn.mixture.GaussianMixture.html#sklearn.mixture.GaussianMixture if it has array API support or not. Having to go to https://scikit-learn.org/stable/modules/array_api.html#support-for-array-api-compatible-inputs is possible but a bit tedious.

Until today I was thinking that we don't need this because "all estimators will support array API input, even if it doesn't make them faster". Which removes the need to explicitly list it on each estimator's page.

What do others think about this?

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Moved to [31](https://github.com/godofecht/flow-scikit/issues/31) in the flow-scikit repository. Further library-specific triage and implementation will happen there.

**Comment by godofecht** [comment]
Moved to [30](https://github.com/godofecht/flow-scikit/issues/30) in the flow-scikit repository. Further library-specific triage and implementation will happen there.
